console/src/main/java/org/nervos/ckb/example/TransactionExample.java

```diff
@@ -0,0 +1,142 @@
+package org.nervos.ckb.example;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+import org.nervos.ckb.address.AddressUtils;
+import org.nervos.ckb.methods.response.CkbTransactionHash;
+import org.nervos.ckb.methods.type.*;
+import org.nervos.ckb.service.CKBService;
+import org.nervos.ckb.service.HttpService;
+import org.nervos.ckb.system.SystemContract;
+import org.nervos.ckb.system.bean.CkbSystemContract;
+import org.nervos.ckb.utils.Network;
+import org.nervos.ckb.utils.Numeric;
+
+class TransactionExample {
+
+  private static String MIN_CAPACITY = "6000000000";
+  private static final String NODE_URL = "http://localhost:8114";
+
+  private Script inputLockScript;
+  private String privateKeyHex;
+  private CKBService ckbService;
+
+  public TransactionExample(String privateKeyHex, Script inputLockScript) {
+    this.privateKeyHex = privateKeyHex;
+    this.inputLockScript = inputLockScript;
+    ckbService = CKBService.build(new HttpService(NODE_URL));
+  }
+
+  public String sendCapacity(List<ReceiverBean> receiverBeanList) {
```

Comment:
Maybe `throw Exception` will be more better.

Reply:
ok